services/frontend-v3/src/components/profileForms/personalGrowthForm/PersonalGrowthFormView.jsx

```diff
@@ -157,6 +158,14 @@ const PersonalGrowthFormView = ({
     },
   };
 
+  /* Component Rules for form fields (commented out as example) */
+  // const Rules = {
+  //   required: {
+  //     required: true,
+  //     message: <FormattedMessage id="profile.rules.required" />,
+  //   },
+  // };
+
```

Comment:
Should probably remove this

Reply:
The reason I kept it in there was so it can be used as reference. Currently those forms dont have validation but if they would need it in the future then it must be uncommented. I was just trying to be "consistent" across all forms. I was on the fence about just removing it.

Reply:
I personally don't like future proofing code, in case that feature will never get done - unless you know this is something we'll for sure need in the non-distant future, we should probably remove it. I think the other forms give a good enough example on how to enable rules

Reply:
I would even go a step further, to remove code duplication, to have a single "Rules" object for all of our forms, instead of having to need to redefine it in every form

Reply:
^ will do this in another PR